packages/plugin-ext/doc/how-to-add-new-custom-plugin-api.md

```diff
@@ -89,7 +89,6 @@ import { FooExtImpl } from './foo-ext-impl';
 import * as fooBarAPI from '@bar/foo';
 
 type FooBarApi = typeof fooBarAPI;
-type Foo = FooBarApi['Foo'];
 
 const FooBarApiFactory = Symbol('FooBarApiFactory');
 type FooBarApiFactory = ApiFactory<FooBarApi>;
```

Comment:
Similarly to the GotD example, this type definition is actually unused. We can remove it and the `ApiFactory` import as in GotD.

Reply:
Above you pointed out it's used at line 132, and that does appear to be the case. So am I wrong about being wrong?

Reply:
I guess part of the confusion is that both snippets are actually in the same file, and thus the contents of the file are spread out in two disjointed parts of the documentation.

Reply:
So, where I got confused is that I mis-associated your comment here. My eye went to the pink-highlighted line (92) instead of the line directly above your comment (95). I should probably get more coffee. All is good. I'll fix the one issue

Reply:
And we also need to remove the now-unused import of `ApiFactory` from the Foo example in the MD as we did for the GotD example in "real code".

Reply:
Indeed. Done